spec/requests/api/v1/apps/credentials_spec.rb

```diff
@@ -27,6 +27,8 @@
             client_id: token.application.uid
           )
         )
+
+        expect(body_as_json[:client_secret_expires_at]).to_not be_present
```

Comment:
I'm undecided as to if this value should or should not be returned here.

Reply:
@ClearlyClaire any opinion on this?

Reply:
The use case for it might be to provide an upgrade path, but as we're currently not expiring clients, just indicating that we **don't** expire clients (anymore) it's a bit hard to judge the utility